examples/gnrc_border_router/Makefile

```
@@ -62,6 +62,14 @@ else ifeq (uhcp,$(PREFIX_CONF))
   USEMODULE += gnrc_uhcpc
 else ifeq (auto_subnets,$(PREFIX_CONF))
   USEMODULE += gnrc_ipv6_auto_subnets_simple
+else ifeq (static,$(PREFIX_CONF))
+  IPV6_ADDR ?= 2001:db8:1::1
+  IPV6_GATEWAY ?= fe80::1
```

Forgot rename?
```suggestion
  IPV6_DEFAULT_ROUTER ?= fe80::1
```

or

```suggestion
  IPV6_DR ?= fe80::1
```